HOW_TO_RELEASE.md

```diff
@@ -15,8 +15,14 @@ upstream        https://github.com/pydata/xarray (push)
 
  1. Ensure your master branch is synced to upstream:
      ```sh
+     git switch master
```

Comment:
is this a new feature? The version of git I'm using only has `checkout`

Reply:
Looks like it's 2.23, last August! Too new?

Reply:
yeah, well, I've got 2.20.1, which is the version included in debian stable (released in July 2019). Nothing too serious, I guess, maintainers should know how to switch branches.

Reply:
OK!